src/main/java/org/folio/completablefuture/FolioVertxCompletableFuture.java

```diff
@@ -15,6 +15,16 @@
 import io.vertx.core.Vertx;
 
 /**
+ *  * NOTICE:
+ *  <p>
+ *  Original class <a href="https://github.com/cescoffier/vertx-completable-future/blob/master/src/main/java/me/escoffier/vertx/completablefuture/VertxCompletableFuture.java">VertxCompletableFuture</a>
```

Comment:
It seems that the copy has been taken from the vertx-completable-future-0.1.2 tag, not from master.
Then the link should be
https://github.com/cescoffier/vertx-completable-future/blob/vertx-completable-future-0.1.2/src/main/java/me/escoffier/vertx/completablefuture/VertxCompletableFuture.java